scala/build.sbt

```diff
@@ -0,0 +1,26 @@
+name := "getting_started"
+
+version := "1.0.0"
+
+scalaVersion := "2.11.6"
+
+scalacOptions ++= Seq("-feature", "-language:higherKinds")
+
+libraryDependencies ++= Seq(
+  "io.concord" % "concord" % "0.1.0",
+  "io.concord" % "rawapi" % "0.1.0"
+)
+
+
+assemblyMergeStrategy in assembly := {
+  case x if x.endsWith("project.clj") => MergeStrategy.discard // Leiningen build files
```

Comment:
What if we have 1 multiproject sbt ? I guess no need - > thoughts ? 

/cc

@RBlafford @bigs